### PR TITLE
chore: refactor KVS

### PR DESCRIPTION
**Pull request by vdusek**
- TODO: 
  - merge: `schemas/keyValueStore/keyValueStore.yaml` + `schemas/KVStore.yaml`?
- edit: transforming this into a draft for now

### CODE REVIEW

File: openapi/components/schemas/keyValueStore/keyValueStore.yaml [hunk 1]
```diff
@@ -36,3 +36,5 @@ properties:
   actRunId:
     type: string
     nullable: true
+  stats:
+    $ref: ./keyValueStoreStats.yaml
```
Comment (tobice) [inline openapi/components/schemas/keyValueStore/keyValueStore.yaml:40]: (nit) I'd use capital K. Same on this file and others as well.

File: openapi/components/schemas/GetListOfKeysResponse.yaml
Comment (tobice) [inline openapi/components/schemas/GetListOfKeysResponse.yaml:1]: (nit) Pls move to a subfolder

Comment (Jkuzz) [reply openapi/components/schemas/GetListOfKeysResponse.yaml:1]: Moved all relevant files 👍

File: openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-title: Getlistofkey-valuestoresresponse
+title: CreateKeyValueStoreResponse
```
Comment (Jkuzz) [inline openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml:1]: No idea why, but git is mixing up these three files. 🤦 

`GetListOfKeysResponse` -> `UpdateStoreResponse` -> `CreateKeyValueStoreResponse`

Comment (tobice) [reply openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml:1]: :sweat_smile:

File: openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 required:
   - data
 type: object
 properties:
   data:
     allOf:
-      - $ref: ./data11.yaml
+      - $ref: ./KeyValueStore.yaml
```
Comment (tobice) [inline openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml:8]: Btw just learned that if you are not combining multiple definitions together, `allOf` is not needed.

```suggestion
properties:
  data:
      $ref: ./KeyValueStore.yaml
```

No need to fix this, just for the next time.

**Review by tobice — APPROVED** [review]

**Pull request merged**